conu/backend/docker/container.py

```diff
@@ -475,3 +490,70 @@ def write_to_stdin(self, message):
             self.popen_instance.stdin.flush()
         except subprocess.CalledProcessError as e:
             raise ConuException(e)
+
+    def get_container_metadata(self):
+        """
+        Convert dictionary returned after docker inspect command into instance of ContainerMetadata class
+        :return: ContainerMetadata, container metadata instance
+        """
+
+        docker_metadata = self.get_metadata(refresh=True)
+
+        # format of Environment Variables from docker inspect:
+        # ['DISTTAG=f26container', 'FGC=f26']
+        env_variables = dict()
+        for env_variable in docker_metadata['Config']['Env']:
+            try:
+                env_variables.update({env_variable.split('=', 1)[0]: env_variable.split('=', 1)[1]})
+            except IndexError:
+                pass  # wrong format of environment variable
+
+        # format of image name from docker inspect:
+        # sha256:8f0e66c924c0c169352de487a3c2463d82da24e9442fc097dddaa5f800df7129
+        image = Image(docker_metadata['Image'].split(':')[1])
+
+        status = ContainerStatus.CREATED
+
+        if docker_metadata['State']['Status'] == 'created':
```

Comment:
This could be a method on ContainerStatus -> `status = ContainerStatus.get(string)`